plan-patches/prometheus-lb-aws/README.md

```diff
@@ -16,12 +16,10 @@ You'll be able to reach grafana, alertmanager, and prometheus on the load balanc
    ```bash
    git clone https://github.com/bosh-prometheus/prometheus-boshrelease.git
    bosh deploy -d prometheus prometheus-boshrelease/manifests/prometheus.yml \
-     -o add-nginx-network-properties.yml
+     -o nginx-vm-extension.yml
```

Comment:
could you fix this up so it's copy+pastable? the path should reach into prometheus-bosh-release/...

thanks!